tfhe/src/c_api/high_level_api/u2048.rs

```diff
@@ -0,0 +1,88 @@
+use crate::c_api::utils::*;
+use std::os::raw::c_int;
+
+#[derive(Copy, Clone)]
+#[repr(C)]
+pub struct U2048 {
+    words: [u64; 32],
+}
+
+impl From<crate::integer::bigint::U2048> for U2048 {
+    fn from(value: crate::integer::bigint::U2048) -> Self {
+        Self { words: value.0 }
+    }
+}
+
+impl From<U2048> for crate::integer::bigint::U2048 {
+    fn from(value: U2048) -> Self {
+        Self(value.words)
+    }
+}
+
+/// Creates a U2048 from little endian bytes
+///
+/// len must be 256
```

Comment:
when/where is it checked ?

Reply:
deep down in the call satck https://github.com/zama-ai/tfhe-rs/blob/6014968655bb3f283d6caaca5ac90500a1d5ae78/tfhe/src/integer/bigint/algorithms.rs#L9

Reply:
alright, those could likely use some nice error message, but it's not required for this